dozer-types/src/types/field.rs

```diff
@@ -536,18 +737,15 @@ impl Field {
     pub fn to_json(&self) -> Option<JsonValue> {
         match self {
             Field::Json(b) => Some(b.to_owned()),
-            Field::UInt(u) => Some(JsonValue::Number(OrderedFloat((*u) as f64))),
-            Field::U128(u) => Some(JsonValue::Number(OrderedFloat((*u) as f64))),
-            Field::Int(i) => Some(JsonValue::Number(OrderedFloat((*i) as f64))),
-            Field::I128(i) => Some(JsonValue::Number(OrderedFloat((*i) as f64))),
-            Field::Float(f) => Some(JsonValue::Number(*f)),
-            Field::Boolean(b) => Some(JsonValue::Bool(*b)),
-            Field::String(s) => match JsonValue::from_str(s.as_str()) {
-                Ok(v) => Some(v),
-                _ => None,
-            },
-            Field::Text(t) => Some(JsonValue::String(t.to_owned())),
-            Field::Null => Some(JsonValue::Null),
+            Field::UInt(u) => Some((*u).into()),
+            Field::U128(u) => Some((*u as f64).into()),
+            Field::Int(i) => Some((*i).into()),
+            Field::I128(i) => Some((*i as f64).into()),
+            Field::Float(OrderedFloat(f)) => Some((*f).into()),
+            Field::Boolean(b) => Some((*b).into()),
+            Field::String(s) => json_from_str(s.as_str()).ok(),
+            Field::Text(t) => Some(t.into()),
```

Comment:
Should this go through `json_from_str`?

Reply:
I'm not sure. The previous implementation had 
```rust
            Field::String(s) => match JsonValue::from_str(s.as_str()) {
                Ok(v) => Some(v),
                _ => None,
            },
            Field::Text(t) => Some(JsonValue::String(t.to_owned())),
```
which makes sense in its own way. WDYT?

Reply:
I'm not sure either. Let's keep the behaviour then.